Storage/ThirdParty/SwiftData.swift

```diff
@@ -132,7 +153,8 @@ open class SwiftData {
         self.schema = schema
         self.files = files
 
-        self.sharedConnectionQueue = DispatchQueue(label: "SwiftData queue: \(filename)", attributes: [])
+        self.primaryConnectionQueue = DispatchQueue(label: "SwiftData primary queue: \(filename)", attributes: [])
```

Comment:
explicitly specifying a `.serial` or `.concurrent` would help clarity. I assume this is `.serial`, because I know our individual DB queues are serial.

Reply:
I agree this sucks because of lack of clarity. However, there's no such `.serial` value here. The empty `DispatchQueue.Attributes` value is the way to specify "serial":

https://developer.apple.com/documentation/dispatch/dispatchqueue/attributes

Reply:
Oh I recall that now, I think I make this same comment every time anyone touches this code. Odd API!